src/test/test-execute.c

```diff
@@ -265,6 +267,16 @@ static void test_exec_workingdirectory(Manager *m) {
         (void) rm_rf("/tmp/test-exec_workingdirectory", REMOVE_ROOT|REMOVE_PHYSICAL);
 }
 
+static void test_exec_execsearchpaths(Manager *m) {
+        assert_se(mkdir_p("/tmp/test-exec_execsearchpaths", 0755) >= 0);
+
+        assert_se(copy_file("/bin/ls", "/tmp/test-exec_execsearchpaths/ls_temp", 0, 0777, 0, 0, COPY_REPLACE) == 0);
```

Comment:
we usually check for success of our functions with >= 0 universally. i.e. we have the rule < 0 indicates error, >= 0 indicates success.